static/app/actionCreators/pageFilters.tsx

```diff
@@ -50,10 +50,6 @@ type Options = {
    * Persist changes to the page filter selection into local storage
    */
   save?: boolean;
-  /**
-   * Skip checking desync state after updating the page filter value.
-   */
-  skipDesyncUpdate?: boolean;
```

Comment:
This is no longer needed — we'll always skip desync updates (unless the site has just been loaded).